src/main/java/org/skriptlang/skript/misc/colors/ExprBlend.java

```diff
@@ -0,0 +1,95 @@
+package org.skriptlang.skript.misc.colors;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.Color;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Blended Colours")
+@Description({
+	"Returns the result of blending colours together. Optionally takes an amount to blend the colours by, which is" +
+		"a number from 0 to 100.",
+	"In that range, a 50 would be an expected equal blend of each colour (the default behaviour)."
+})
+@Examples({
+	"set {_purple} to red blended with blue",
+	"set {_goldyPurple} to {_purple} blended with gold",
+	"set {_morePurpleThanGold} to {_purple} blended with gold by an amount of 10",
+	"set {_aBunch} to red blended with all colours where [input is not red]"
+})
+@Since("INSERT VERSION")
+public class ExprBlend extends SimpleExpression<Color> {
+
+	static {
+		Skript.registerExpression(ExprBlend.class, Color.class, ExpressionType.COMBINED,
+			"%colors% (blended|mixed) with %colors% [by [a[n] (factor|amount) of] %-number%]",
+			"blend of %colors% (and|with) %colors% [by [a[n] (factor|amount) of] %-number%]",
+			"%colors% and %colors% blen(ded|t) together [by [a[n] (factor|amount) of] %-number%]");
```

Comment:
```suggestion
			"%colors% (blended|mixed) with %colors% [by [(a factor|an amount) of] %-number%]",
			"blend of %colors% (and|with) %colors% [by [(a factor|an amount) of] %-number%]",
			"%colors% and %colors% blen(ded|t) together [by [(a factor|an amount) of] %-number%]");
```
I think this is somewhat easier to read (and a bit more correct!)